migrator/src/main/resources/log4j.properties

```diff
@@ -11,7 +11,7 @@ log4j.rootLogger=INFO, console
 
 # LOG FILE LOCATIONS
 log4j.appender.console=org.apache.log4j.ConsoleAppender
-log4j.appender.console.Threshold=INFO
+log4j.appender.console.Threshold=DEBUG
 log4j.appender.console.layout=org.apache.log4j.PatternLayout

```

Comment:
No, that will be reverted back. I set that to debug to see the test logs.

On Thu, May 14, 2015 at 10:36 AM, Dennis Crissman <notifications@github.com>
wrote:

> In migrator/src/main/resources/log4j.properties
> <https://github.com/lightblue-platform/lightblue-migrator/pull/191#discussion_r30336885>
> :
>
> > @@ -11,7 +11,7 @@ log4j.rootLogger=INFO, console
> >
> >  # LOG FILE LOCATIONS
> >  log4j.appender.console=org.apache.log4j.ConsoleAppender
> > -log4j.appender.console.Threshold=INFO
> > +log4j.appender.console.Threshold=DEBUG
> >  log4j.appender.console.layout=org.apache.log4j.PatternLayout
>
> Do we want to change the log level?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-migrator/pull/191/files#r30336885>
> .
>